### ISSUE TITLE
Input fields in cart/checkout forms are read twice with a screen reader

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1775]
![imatge](https://user-images.githubusercontent.com/3616980/74833401-ae31ab80-5319-11ea-843d-817285690172.png)

**To Reproduce**
1. Navigate the _Checkout_ block with a screen reader and the arrow keys (instead of <kbd>Tab</kbd>).
2. Notice every input field is read twice: one for the input field and one for the label. That makes navigating with a screen reader very noisy, because every field is read twice.

This is also happening in the _Checkout_ shortcode. So it's not a regression in blocks, but still would be nice to improve the experience.

**Expected behavior**
Each field should be read only once.

**Comment by mikejolley** [comment]
This doesn't happen with chrome and voiceover. Any steps to reproduce this @Aljullu ?

**Comment by Aljullu** [comment]
Might be an issue with Orca (screen reader for Linux), I think it doesn't play well with `aria-label` on input fields or something like that. I remembered we had a similar case that only I could reproduce.

Since I couldn't spot any issue in the markup when I filled this, I think it's safe to close. Thanks for testing in Mac @mikejolley!
